File: Lancetilla BackEnd/Lancetilla/Lancetilla.API/Models/AlimentacionViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Lancetilla.API.Models
{
    public class AlimentacionViewModel
    {

        public int alim_Id { get; set; }
        public string alim_Descripcion { get; set; }
        public int? alim_UserCreacion { get; set; }
        
        public int? alim_UserModificacion { get; set; }
      
    }
}

File: Lancetilla FrontEnd/src/app/demo/Models/AlimentacionViewModel.ts
export class AlimentacionViewModel  {

    constructor(
        public alim_Id?: number,
        public alim_Descripcion?:string,
        public usua_UserCreaNombre?:string,
        public alim_UserCreacion?: 1,
        public alim_FechaCreacion?:string,
        public usua_UserModiNombre?: number,
        public alim_UserModificacion?: number,
        public alim_FechaModificacion?: string,
        public alim_Estado?: boolean
    ){}

}